powheg_gridpack.py

```diff
@@ -33,21 +39,21 @@ def get_run_card(self):
                                              dataset_dict.get('model_params_user', []),
                                              model_params_vars)
 
-        return run_card + '\n' + customize_card
+        return customize_card
 
-    def prepare_run_card(self):
+    def prepare_input_card(self):
         """
-        Get run card and write it to job files dir
+        Get run card and customize card, glue them together into input card and write it to job files dir
         """
         job_files_path = self.get_job_files_path()
         output_file_name = os.path.join(job_files_path, 'powheg.input')
-        run_card = self.get_run_card()
-        self.logger.debug('Writing customized run card %s', output_file_name)
-        self.logger.debug(run_card)
+        input_card = self.get_run_card() + '\n\n\n\n' + self.get_customize_card()
+        self.logger.debug('Writing customized input card %s', output_file_name)
+        self.logger.debug(input_card)
         with open(output_file_name, 'w') as output_file:
-            output_file.write(run_card)
+            output_file.write(input_card)
```

Comment:
Rename `run_card` to `input_card` following powheg input convention `powheg.input`
Here `run_card` and `customize_card` are read, glued, and written to powheg.input